tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/ensure_date_datetime_consistency_test.py

```diff
@@ -140,8 +142,23 @@ def test_required_field_datetime_pairs(self):
           (108, 888888, 0, date('2016-05-01'), timestamp('2016-05-01 11:00:00'), 0, date('2016-05-02'), timestamp('2016-04-15 11:00:00'))
         """)
 
-        query = tmpl.render(fq_dataset_name=self.fq_dataset_name)
-        self.load_test_data([query])
+        note_tmpl = self.jinja_env.from_string("""
+                INSERT INTO `{{fq_dataset_name}}.note`
+                (note_id, person_id, note_date, note_datetime, note_type_concept_id, note_class_concept_id,
+                 note_title, note_text, encoding_concept_id, language_concept_id)
+                VALUES
+                  (101, 222222, date('2016-05-01'), timestamp('2016-05-01 11:00:00'), 0, 0, '', '', 0, 0 ),
+                  (102, 333333, date('2016-05-01'), timestamp('2016-05-07 11:00:00'), 0, 0, '', '', 0, 0),
+                  (103, 444444, date('2016-05-01'), timestamp('2015-05-07 11:00:00'), 0, 0, '', '', 0, 0),
+                  (104, 555555, date('2016-05-01'), timestamp('2016-05-01 11:00:00'), 0, 0, '', '', 0, 0),
+                  (105, 666666, date('2016-05-01'), timestamp('2016-05-01 11:00:00'), 0, 0, '', '', 0, 0),
+                  (106, 777777, date('2016-05-01'), timestamp('2016-05-01 11:00:00'), 0, 0, '', '', 0, 0),
+                  (107, 888888, date('2016-05-01'), timestamp('2016-05-01 11:00:00'), 0, 0, '', '', 0, 0)
+                """)
+        condition_query = condition_tmpl.render(
+            fq_dataset_name=self.fq_dataset_name)
+        note_query = note_tmpl.render(fq_dataset_name=self.fq_dataset_name)
+        self.load_test_data([condition_query, note_query])
```

Comment:
The test for `note` really only needs to have two entries.  This can be simplified with the given patch.  
[ensure_date_datetime_consistency_test_patch.txt](https://github.com/all-of-us/curation/files/6804094/ensure_date_datetime_consistency_test_patch.txt)
This patch tests the two fields in `note` to with  "control" and "expected change" records.